### PR TITLE
Edit/ddrtv 10 permission management

### PR DESCRIPTION
**Pull request by graz-dev**
## Description

Rewrite permission management docs to improve readability and clearness

## Pull Request Type

- [X] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [X] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [X] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [X] No sensible content has been committed

### CODE REVIEW

File: docs/development_suite/identity-and-access-management/console-levels-and-permission-management.md [hunk 1]
```diff
@@ -3,22 +3,24 @@ id: console-levels-and-permission-management
...
-At this level you can configure resources such as Companies, Project Templates and Marketplace.
+1. **Console Level** is the root level regarding the general configuration of your entire Console. At this level you can configure resources such as Companies, Project Templates and Marketplace.
 
-1. **Company level** is the second hierarchical level. A Company can contain several underlying Projects, which can inherit different kinds of information from the Company without needing further configuration. At this level, License, Clusters, Providers, Users and Service Accounts can be managed. To know more about Company configuration, check out the dedicated [documentation section](/development_suite/company/create.md).
+1. **Company Level** is the second hierarchical level. A Company (aka Tenant) can contain several underlying Projects, which can inherit different kinds of information from the Company without needing further configuration. At this level, License, Clusters, Providers, Users and Service Accounts can be managed. To know more about Company configuration, check out the dedicated [documentation section](/development_suite/company/create.md).
```
[the remark this answers is not in the document]
Comment (fredmaggiowski) [reply docs/development_suite/identity-and-access-management/console-levels-and-permission-management.md:16]: This is not correct; tenants do not exist anymore; they are named Companies throughout the documentation and the product

File: docs/development_suite/identity-and-access-management/console-levels-and-permission-management.md [hunk 2]
```diff
@@ -3,22 +3,24 @@ id: console-levels-and-permission-management
 title:  Console levels and permission management
 sidebar_label: Console Levels and Permission Management
 ---
-Assigning different roles and permissions to each [identity](/development_suite/identity-and-access-management/overview.md#identity-and-access-management) that has access to your resources is a key action for defining responsibilities within your Platform ecosystem.
-Mia-Platform Console is based on hierarchical levels and, for each of them, specific permissions and capabilities have been identified and can be assigned.  
+Having the ability to assign specific permissions and roles to each [Identity](/development_suite/identity-and-access-management/overview.md#identity-and-access-management) that has access to the system in a detailed way is a key point in defining responsibilities within your platform ecosystem.
```
[the remark this answers is not in the document]
Comment (fredmaggiowski) [reply docs/development_suite/identity-and-access-management/console-levels-and-permission-management.md:6]: Identities are references with the Capital *I* elsewhere in the doc, I have no particular preference but let's be coherent if we want to change this

[the remark this answers is not in the document]
Comment (graz-dev) [reply docs/development_suite/identity-and-access-management/console-levels-and-permission-management.md:6]: I suggest to leave it with the capital letter. 
@akelity if you think that the usage of the capital letter is wrong, I can review the other documentation pages to be consistent.
Let me know what to do.

[the remark this answers is not in the document]
Comment (akelity) [reply docs/development_suite/identity-and-access-management/console-levels-and-permission-management.md:6]: ack

**Pull request merged**